### PR TITLE
[rebuild]userprofile

### PR DESCRIPTION
**Pull request by joshuaobinna**

### CODE REVIEW

File: src/components/userProfile/components/styles.css [hunk 1]
```diff
@@ -1,86 +1,72 @@
...
+    color: darkgray;
+    font-weight: 400;
 }
-/**UserProfile.js */
-.big-content {
-    background: white;
+
+/* container for avatar and photo edit botton in userInfo.js */
+.avatar-container {
+    background-color: white !important;
```
Comment (EngrEric) [inline src/components/userProfile/components/styles.css:31]: why are you using important here?

**Review by EngrEric — CHANGES_REQUESTED** [review]
Looks good to me.
remember to bump version

File: src/components/userProfile/constants.js [hunk 2]
```diff
@@ -1,72 +1,57 @@
...
+    key: "contact",
+}, {
+    heading: "LOCATION",
+    info: [
+        { description:  '24, Wilkie Crescent, Ikeja', title: "Address:" },
+        { description: "Lagos", title: 'City:' },
+        { description: "Nigeria", title: 'Country:' }, 
+    ],
+    key: "location",
+},
```
Comment (Way2nnadi) [inline src/components/userProfile/constants.js:54]: why are you grabbing the basic information from a static file. Get the user information from redux

**Review by Way2nnadi — CHANGES_REQUESTED** [review]
@joshuaobinna the user information already lives in the redux store, grab it from there and display it accordingly

File: src/components/userProfile/constants.js (hunk 2 above)
Comment (joshuaobinna) [reply src/components/userProfile/constants.js:54]: This I hsve another PR for that. This is PR is just to changer the visual look of the user profile

File: src/components/userProfile/components/styles.css (hunk 1 above)
Comment (joshuaobinna) [reply src/components/userProfile/components/styles.css:31]: I'll look into it

**Comment by EngrEric** [comment]
> @joshuaobinna the user information already lives in the redux store, grab it from there and display it accordingly

@enyeInc the idea is to have better look for the user profile. Populating the interface with the actual user data is for another PR; this is because we want to restructure how the user data is sent to the App component. hence integrating auth0 is changing some of the app flow

**Comment by enyeInc** [comment]
@EngrEric and @joshuaobinna gotcha

**Review by enyeInc — APPROVED** [review]

**Review by enyeInc — APPROVED** [review]

File: src/components/userProfile/components/UserInfo.js [hunk 3]
```diff
@@ -1,49 +1,64 @@
...
+                <div className={AVATAR_CONTAINER}>
+                    <Avatar
+                        className={AVATAR}
+                        src={PROFILE_PIX}
+                        shape={SQUARE}
+                    />
+                    <Button type={GHOST} size={SMALL} className={EDIT_BOTTON}>
+                        {EDIT_PHOTO}
+                        <Icon type={EDIT} />
+                    </Button>
```
Comment (Simpcyclassy) [inline src/components/userProfile/components/UserInfo.js:59]: image
I actually think the edit button will look better as just an icon on top of the image maybe by the right or just beside the image

**Review by Simpcyclassy — APPROVED** [review]
Every other thing looks good to me apart from the `Edit` button preference

**Review by EngrEric — APPROVED** [review]

File: src/components/userProfile/components/UserInfo.js (hunk 3 above)
Comment (joshuaobinna) [reply src/components/userProfile/components/UserInfo.js:59]: I think its ok where it is

**Comment by EngrEric** [comment]
@joshuaobinna  please resolve the conflict and merge down this PR

**Pull request merged**